src/common/balances.lua

```diff
@@ -1,54 +1,81 @@
+--- Module for managing balances and transactions.
+-- @module balances
+
 local utils = require(".common.utils")
-local json = require(".common.json")
 
 local balances = {}
 
-function balances.walletHasSufficientBalance(wallet)
-	return Balances[wallet] ~= nil and Balances[wallet] > 0
-end
-
+--- Transfers the ANT to a specified wallet.
+---@param to string - The wallet address to transfer the balance to.
+---@return nil
 function balances.transfer(to)
 	utils.validateArweaveId(to)
 	Balances = { [to] = 1 }
+	--luacheck: ignore Owner Controllers
 	Owner = to
 	Controllers = {}
-	return json.encode({ [to] = 1 })
 end
 
+--- Retrieves the balance of a specified wallet.
+---@param address string - The wallet address to retrieve the balance from.
+---@return integer - Returns the balance of the specified wallet.
 function balances.balance(address)
 	utils.validateArweaveId(address)
 	local balance = Balances[address] or 0
 	return balance
 end
 
+--- Retrieves all balances.
+---@return table<string, integer> - Returns the encoded JSON representation of all balances.
 function balances.balances()
-	return json.encode(Balances)
+	return Balances
 end
 
+--- Sets the name of the ANT.
+---@param name string - The name to set.
+---@return table<string, string>
 function balances.setName(name)
 	assert(type(name) == "string", "Name must be a string")
 	Name = name
-	return json.encode({ name = Name })
+	return { Name = Name }
```

Comment:
lowercase is fine - upper suggests global state

Reply:
I changed it to uppercase for the relavent APIs because the spec declares it should be thus https://github.com/ar-io/specs/blob/main/arns/arns-token-1.md#set-name